### PR TITLE
Fix: Fix connecting wifi manually bug

### PR DESCRIPTION
**Pull request by jane-rose**

### CODE REVIEW

**Comment by jane-rose** [comment]
Fixes #640

File: src/app/widgets/Connection/WifiConnection.jsx [hunk 1]
```diff
@@ -192,6 +193,7 @@ class WifiConnection extends PureComponent {
 
                         this.props.setManualIP(text);
                         const server = new Server('Manual', text);
+                        this.props.addSelectedServerToList(server);
```
Comment (parachvte) [inline src/app/widgets/Connection/WifiConnection.jsx:196]: Note that we only support **one single** manual server (cause we only save one).
So everytime we call `setSelectedServer()`, the server must be added to server list.

=> `addSelectedServerToList()` action is not necessary, the logic can be put inside of `setSelectedServer()`.

File: src/app/flux/machine/action-connect.js [hunk 2]
```diff
@@ -41,6 +41,22 @@ const setSelectedServer = (server) => (dispatch) => {
     dispatch(baseActions.updateState({ server }));
 };
 
+const addSelectedServerToList = (server) => (dispatch, getState) => {
```
Comment (parachvte) [inline src/app/flux/machine/action-connect.js:44]: The action will cause only `server` present in `servers`, but what we need is prepend selected `server` to server list, not replacing it.

**Review by parachvte — CHANGES_REQUESTED** [review]
inappropriate logic implement, changes needed.

File: src/app/widgets/Connection/WifiConnection.jsx (hunk 1 above)
Comment (jane-rose) [reply src/app/widgets/Connection/WifiConnection.jsx:196]: No, i konw think so.

File: src/app/flux/machine/action-connect.js (hunk 2 above)
Comment (jane-rose) [reply src/app/flux/machine/action-connect.js:44]: Yep, that's make sense

**Pull request merged**